### PR TITLE
Update Window_and_Application_API_Spec.md

### PR DESCRIPTION
**Pull request by thomasclaudiushuber**
I don't know what the property should be called there, but definitely not Visible. :-)

### CODE REVIEW

File: active/Win32/Window_and_Application_API_Spec.md [hunk 1]
```diff
@@ -482,9 +482,9 @@ Contains the argument returned by a window size change event.
     public bool Handled { get; set; };
     ```
 
-- WindowSizeChangedEventArgs.Visible: Gets the new size of the window in units of effective (view) pixels.
+- WindowSizeChangedEventArgs.WindowSize: Gets the new size of the window in units of effective (view) pixels.
     ```CS
-    public Size Visible { get; } 
+    public Size WindowSize { get; }
```
Comment (dotMorten) [inline active/Win32/Window_and_Application_API_Spec.md]: I'd prefer just `Size`

Comment (thomasclaudiushuber) [reply active/Win32/Window_and_Application_API_Spec.md]: Yes, I thought about that too. But maybe when you have an event handler and the parameter is just called "e", e.WindowSize looks better than e.Size. But I'm not an API design expert. :-)

**Comment by dotMorten** [comment]
Then the other size changed event should be called ControlSize. It's more consistent this way.
Also don't see why we can't reuse the existing SizeChangedEventArgs class here?

File: active/Win32/Window_and_Application_API_Spec.md (hunk 1 above)
Comment (dotMorten) [reply active/Win32/Window_and_Application_API_Spec.md]: Then the other size changed event should be called ControlSize. It's more consistent just calling it Size.
Also don't see why we can't reuse the existing SizeChangedEventArgs class here?

Comment (thomasclaudiushuber) [reply active/Win32/Window_and_Application_API_Spec.md]: Ok, changed to just Size.

Yes, the question with SizeChangedEventArgs is good. @marb2000 might know. :)

**Pull request merged**